sql/core/src/test/resources/sql-tests/inputs/pgSQL/text.sql

```diff
@@ -0,0 +1,134 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- TEXT
+-- https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/text.sql
+
+SELECT string('this is a text string') = string('this is a text string') AS true;
+
+SELECT string('this is a text string') = string('this is a text strin') AS `false`;
+
+CREATE TABLE TEXT_TBL (f1 string) USING parquet;
+
+INSERT INTO TEXT_TBL VALUES ('doh!');
+INSERT INTO TEXT_TBL VALUES ('hi de ho neighbor');
+
+SELECT '' AS two, * FROM TEXT_TBL;
+
+-- As of 8.3 we have removed most implicit casts to text, so that for example
+-- this no longer works:
+-- Spark SQL implicit cast integer to string
+select length(42);
```

Comment:
FYI: If we strictly follow ANSI/SQL, we don't allow this implicit cast along with PostgresSQL.
cc: @gengliangwang

Reply:
Is this casting an integer to string? If yes, I think it is allowed in ANSI SQL and up-cast.